app/javascript/controllers/sortable_list_controller.js

```diff
@@ -0,0 +1,13 @@
+import { Controller } from "@hotwired/stimulus"
```

Comment:
There's some strange behaviour when trying to sort a list which is scrollable. When reordering element `A` from the top to the bottom of the list, the scrolling only works if you center the list item correctly in the div and then try to move it. Looks like we should be able to get this working correctly by using Autoscroll. See https://github.com/SortableJS/Sortable/blob/master/plugins/AutoScroll/README.md

Reply:
Great catch, added in 2c820683116e72604aea8b4d070622519399006a.
Let me know if the scroll still doesn't feel good, we can adjust the speed and how far from the div edge the element has to be before scrolling begins

Reply:
This is working great!